python/apps/MeasureDelay.py

```diff
@@ -1,186 +1,227 @@
-########################################################################
-## Measure round trip delay through RF loopback/leakage
-########################################################################
+#!/usr/bin/env python
+"""Measure round trip delay through RF loopback/leakage
 
-import SoapySDR
-from SoapySDR import * #SOAPY_SDR_ constants
```

Comment:
FYI this doesnt pollute the namespace or anything because the module defines `__all__` to just contain the define style constants. Not sure if its a style thing, but its safe to do this import without strange side-effects. Ex: `python -c "from SoapySDR import *; print(globals())"`